worldengine/cli/main.py

```diff
@@ -335,9 +336,24 @@ def main():
                                action="store_true",
                                help="Draw outer land border",
                                default=False)
-
     # TODO: allow for RGB specification as [r g b], ie [0.5 0.5 0.5] for gray
 
+    # -----------------------------------------------------
+    export_options = parser.add_argument_group(
+        "Export Options", "You can specify the formats you wish the generated output to be in. ")
+    export_options.add_argument("--export-type", dest="export_type",
+                                help="Export to a specific format such as: BMP or PNG",
+                                default="bmp")
+    export_options.add_argument("--export-bpp", dest="export_bpp", type=int,
+                                help="Bits per pixel: 8, 16 and 32",
+                                default=8)
+    export_options.add_argument("--export-signed", dest="export_signed", action="store_true",
+                                help="Used signed bits or not.",
```

Comment:
Hmm, that's a fair point. While I'm absolutely abysmal with C++, C is a different story; well written C is usually fairly simple, though slow, for me to grasp. I'll look that those sometime soon.